README.md

```diff
@@ -276,6 +276,99 @@ For more information about the image delete command see the help message:
 $ azure-img-utils image delete --help
 ```
 
+## galleryimg command
+
+Group of commands for gallery image management.
+
+### exists subcommand
+
+This subcommand allows the user to check if an image version exists in a
+ gallery.
+The subcommand is *azure-img-utils galleryimg exists*.
```

Comment:
Although the command will be quite long I think we need to be clear on what the sub-command is for. Gallery Images are a different topic. Here we are only handling Gallery Image Versions. Thus I'd suggest the sub-command like gallery-img-versions or gallery-image-versions.